src/Sulu/Bundle/CoreBundle/Entity/ApiEntity.php

```diff
@@ -81,7 +81,9 @@ public function createSelfLink()
         // if no apiPath is not set generate it from basepath
         if (\is_null($this->getApiPath())) {
             $class = \explode('\\', \get_class($this));
-            $plural = Inflector::pluralize(\strtolower(\end($class)));
+            // this is deprecated already so no use to do fancy to get the factory
+            $inflector = InflectorFactory::create()->build();
```

Comment:
Would remove this comment does more confuse me as it helps 🙈 

```suggestion
            $inflector = InflectorFactory::create()->build();
```